roles/zooplus/tasks/globalprotect.yml

```
@@ -0,0 +1,23 @@
+---
+
+- name: Check if globalprotect client is installed
+  stat:
+    path: /usr/bin/globalprotect
+  register: globalprotect_existence
+
+- name: Copy globalprotect package
+  copy:
+    src: files/{{ globalprotect_package }}
+    dest: /home/{{ user }}/SW/{{ globalprotect_package }}
+  when: not globalprotect_existence.stat.exists
+
+- name: Install globalprotect
+  command: "dpkg -i /home/{{ user }}/SW/{{ globalprotect_package }}"
```

Try to avoid the command module. It's "nasty" ;)

You can also do smth. like this.
https://docs.ansible.com/ansible/latest/modules/apt_module.html

```suggestion
  apt:
    deb: /home/{{ user }}/SW/{{ globalprotect_package }}"
```

---

please send me the proper package!!! pleasepleaseplease